generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-ftp/pom.xml

```diff
@@ -59,13 +59,13 @@
     <dependency>
       <groupId>org.apache.ftpserver</groupId>
       <artifactId>ftpserver-core</artifactId>
-      <version>1.1.1</version>
+      <version>${quarkus-cassandra-client.version}</version>
```

Comment:
Ufff...

Reply:
What does that mean? :)

Reply:
Indeed this looks strange. This change is not mine, it's a result of running ` mvn clean process-resources` after changing the property `quarkus-cassandra-client.version` to `1.1.1`.

Reply:
It just tried to match the version of the dependency to an existing property and found the Cassandra version one. It's wrong but not a big deal. So don't worry about that. I'll look into that later.